package.json

```diff
@@ -9,33 +9,32 @@
   "overrides": {
     "semver": "^7.5.3"
   },
+  "engines": {
+    "node": "18.18.2",
+    "yarn": "4.0.2"
+  },
   "scripts": {
     "start": "node build/server.js",
     "dev": "nodemon src/server.ts",
     "build": "tsc -p .",
-    "dev:startdb": "npm run db:dockerup && npm run prisma:migrate && npm run prisma:seed",
+    "dev:startdb": "yarn db:dockerup && yarn prisma:migrate && yarn prisma:seed",
     "db:dockerup": "echo \"🐳✨ Starting docker compose...✨🐳\" && docker compose -f docker/docker-compose-db-dev.yml up -d",
     "db:dockerdown": "echo \"🐋 Stopping docker compose... 🐋\" && docker compose -f docker/docker-compose-db-dev.yml down",
-    "prisma:studio": "npx prisma studio",
-    "prisma:generate": "npx prisma generate",
-    "prisma:migrate": "echo \"🔄️ Running migrations...🔄️\" && npx prisma migrate dev",
-    "prisma:seed": "npx prisma db seed",
+    "prisma:studio": "prisma studio",
+    "prisma:generate": "prisma generate",
+    "prisma:migrate": "echo \"🔄️ Running migrations...🔄️\" && yarn prisma migrate dev",
+    "prisma:seed": "prisma db seed",
     "lint": "eslint . --ext .js,.ts",
     "lint:fix": "eslint . --ext .js,.ts --fix",
     "format": "prettier --write 'src/**/*.ts'",
     "prepare": "husky install",
     "test": "jest"
   },
-  "husky": {
-    "hooks": {
-      "pre-push": ".husky/pre-push"
-    }
-  },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "dependencies": {
```

Comment:
During install I get this message:
```
YN0000: ┌ Post-resolution validation
➤ YN0002: │ backend@workspace:. doesn't provide @types/node (pdeab6), requested by ts-node.
➤ YN0086: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code.
➤ YN0000: └ Completed
```
adding `@types/node` seems to remove the warning, although we may need to specify v18.

Reply:
This can be ignored for now
This has nothing to do with yarn.

Reply:
I have added `@types/node` dev dep